packages/flutter/lib/src/cupertino/magnifier.dart

```diff
@@ -258,7 +258,9 @@ class CupertinoMagnifier extends StatelessWidget {
     this.borderSide =
         const BorderSide(color: Color.fromARGB(255, 232, 232, 232)),
     this.inOutAnimation,
-  });
+    this.magnificationScale = 1,
```

Comment:
Which part of the actual magnifier widget uses this parameter? Currently it looks like it is still unused. Or is this PR still a work in progress?

Reply:
I forgot to push it. My bad.